scalableminds/webknossos
# If uploaded dataset needs conversion, tag user only after conversion success

Hand out mailchimp tag HasUploadedOwnDataset (which triggers an onboarding email) only after successful conversion

### Steps to test:
(I tested this locally, I think for a review you only need to have a look at the code)
- Add logging to mailchimp tagging (e.g. `  def tagUser(user: User, tag: MailchimpTag): Unit = {logger.info(s"tagging as ${tag.toString}")` … in `MailchimpClient.scala`, change mail.logToStdout to true in `application.conf`
- Enable jobs, set up worker
- Upload wkw dataset, tagging should happen immediately after upload
- Uplaod tiff dataset, tagging should happen only after successful conversion job (should come together with the job success email)

### Issues:
- fixes #6672

------
